policy/modules/services/xserver.if

```diff
@@ -125,8 +125,12 @@ template(`xserver_restricted_role',`
 	# Client write xserver shm
 	tunable_policy(`allow_write_xshm',`
 		allow $2 xserver_t:shm rw_shm_perms;
+	')
+
+	tunable_policy(`allow_write_xserver_tmpfs',`
```

Comment:
It seems that this block and the block added below should be 
```
tunable_policy(`allow_write_xshm | allow_write_xserver_tmpfs',`
```
so that shm access works correctly.

Reply:
They are three blocks in total...

I can introduce the change you request, given that X shared memory access is much more intrusive than X server tmpfs rw access.

Reply:
Logic OR is double-pipe as in C syntax, so I have edited that too...